src/hoc/layout/index.tsx

```diff
@@ -15,12 +18,12 @@ function Layout(props) {
     (state: RootState) => state.auth.authenticated
   );
 
-  useEffect(() => {
-    if (isUserAuthenticated) {
-      setIsLogin(true);
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  // useEffect(() => {
```

Comment:
`const [isLogin, setIsLogin] = useState(false);` do we need this?
`const { isUserAuthenticated } = useAuthentication();` do we need this?

Reply:
`const [isLogin, setIsLogin] = useState(false);` is used for logging in and logging out the user for temporary purpose, since we did not have any API for it. But I think we can now remove it since Iffort team have implemented login and logout.

`useAuthentication()` js a custom hook that I created for checking whether the user is authenticated or not and can basically replace the `useSelector` used. I have kept both codes for now:
`const { isUserAuthenticated } = useAuthentication();`
`const isAuthenticate = useSelector(
    (state: RootState) => state.auth.authenticated
  );`
So that we can discuss this point tomorrow.